generator/internal/connection_generator.cc

```diff
@@ -357,6 +388,20 @@ Status ConnectionGenerator::GenerateCc() {
   //  clang-format on
 
   for (auto const& method : methods()) {
+    if (IsBidirStreaming(method)) {
+      CcPrintMethod(
+          method, __FILE__, __LINE__,
+          R"""(  std::unique_ptr<::google::cloud::internal::AsyncStreamingReadWriteRpc<
+      $request_type$,
+      $response_type$>>
+  Async$method_name$() override {
+    auto cq = background_->cq();
+    return stub_->Async$method_name$(cq, absl::make_unique<grpc::ClientContext>());
```

Comment:
Should we avoid giving the `CompletionQueue` a name so as to avoid an extra ref-count inc/dec on its `impl_`?

Reply:
Maybe there is a cast or another trick that I am missing?

Reply:
Ack.  (Aside: If we could change `Async$method_name$` to take its `CompletionQueue` by value, I would vote to do so.)